gulpfile.js

```diff
@@ -35,6 +35,12 @@ function getTask(path, options) {
 	};
 }
 
+/*== version ==*/
+gulp.task('patch-version', getTask('./gulp/utils', {
```

Comment:
Not sure why, but I had to move this top from bottom to prevent a `patch-version` task not defined error.